fibonacci/README.md

```diff
@@ -1,8 +1,8 @@
-# Exercise XX - fibonacci
+# Exercise XX - Fibonacci
```

Comment:
I appreciate you capitalizing the exercises. Can we get the actual exercise number here? Thanks!

Reply:
Done.